app/src/main/java/beepbeep/pixels/feed/FeedViewModel.kt

```diff
@@ -0,0 +1,17 @@
+package beepbeep.pixels.feed
+
+import beepbeep.pixels.feed.list.FeedListViewModel
+
+sealed class FeedAction
+
+class SetAction(val items: List<FeedListViewModel>) : FeedAction()
+object LoadAction : FeedAction()
+
+data class FeedViewModel(val isLoading: Boolean = false,
```

Comment:
wanna make this extend from the google component's `ViewModel`? then it can survive orientation change